File: Backend/tecnpoint/src/main/java/com/techsolutions/tecnpoint/entities/DTO/ConversaDTO.java
package com.techsolutions.tecnpoint.entities.DTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Getter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ConversaDTO {
    Long idConversa;
    UsuarioDTO remetente;
    String mensagem;
    LocalDateTime dataHoraEnvio;
}

File: Desktop/TecnPoint/TecnPoint.Modelos/DTO/ConversaDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.InteropServices;
using System.Text;
using System.Threading.Tasks;

namespace TecnPoint.Modelos.DTO
{
    public class ConversaDTO
    {
        public long idConversa { get; set; }
        public UsuarioDTO remetente { get; set; }
        public string mensagem { get; set; }
        public DateTime dataHoraEnvio { get; set; }
    }
}
